contracts/src/bridge/Inbox.sol

```diff
@@ -132,6 +140,7 @@ contract Inbox is DelegateCallAware, PausableUpgradeable, IInbox {
         onlyAllowed
         returns (uint256)
     {
+        if (_chainIdChanged()) revert L1Forked();
```

Comment:
I think we can offer the `non-replayable` entrypoint and not add the extra gas here

Reply:
Attacker would be able to replay Arb1 transaction to the fork tho, is that a concern?

Reply:
I see, good point.